frontend/src/components/Modal/Modal.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useRef } from "react";
+import "./Modal.scss";
+import Button from "../Button/Button";
+
+const Modal = ({ header, body, footer }) => {
```

Comment:
As i mentioned on trello, we need a property called "onClose" which is a function that will be called if we want to close the modal